scico/test/linop/test_radon_svmbir.py

```diff
@@ -40,6 +42,24 @@ def make_A(im, num_angles, num_channels):
     return A
 
 
+def cg_prox(f, v, λ):
```

Comment:
@bwohlberg @Michael-T-McCann This prox implementation may be useful. I could add this to WeightedSquaredL2Loss and SquaredL2Loss so we have a default even when A is not diagonal.

Reply:
Agreed. Please do so. It should also take `**kwargs` so that `v0` can be specified, and there should also be some way of specifying max iterations and stopping tolerance. Perhaps these should go in the `__init__`? @Michael-T-McCann, thoughts?

Reply:
We have the following candidates for arguments that may want to get passed to the cg-prox:
```python
    x0: JaxArray,
    tol: float = 1e-5,
    atol: float = 0.0,
    maxiter: int = 1000,
    M: Optional[Callable] = None,
```
Perhaps we want to keep it consistent with the SVMBIR-prox and only use 
```python
max_iterations=1000,
stop_threshold=0.001
v0=None
```
with these variable names?

Reply:
@bwohlberg  I started a draft of these changes in 3fdc0cb.

- This uses the cg from the `WeightedSquaredL2Loss` to test against the SVMBIR prox. What do you think?
- It does not use any max_iterations or tolerance arguments yet until we agree how to best do this consistently
- It does not have the prox for the unweighted loss yet. Will do once we agree on the weighted version.